api/src/org/apache/cloudstack/api/response/PodResponse.java

```diff
@@ -55,11 +55,11 @@
 
     @SerializedName("startip")
     @Param(description = "the starting IP for the Pod")
-    private String startIp;
+    private List<String> startIp;
```

Comment:
Do you think this breaks API response?

Reply:
As we are adding multiple ranges in a pod, so there will be a list of startIps and endIps. It is not breaking because if you look at the createPodResponse method of ApiResonseHelper, where the startIp and endIp variable are converted to list. And the Start IP and End IP field are removed from pod detail page. Hope this clarifies.